Dockerfile.s3proxy

```diff
@@ -0,0 +1,48 @@
+FROM golang AS builder
+
+COPY . /rclone/
+WORKDIR /rclone/
+
+RUN \
+  CGO_ENABLED=0 \
+  make && \
+  ./rclone version
+
+
+# Begin final image
+FROM alpine:latest
+
+COPY --from=builder /rclone/rclone /usr/local/bin/
+
+COPY ["./docker/startup", "/startup"]
+
+RUN apk --no-cache add \
+  ca-certificates \
+  fuse3 \
+  tzdata \
+  && apk cache clean
+
+RUN echo "user_allow_other" >> /etc/fuse.conf && \
+  addgroup -g 1009 rclone && \
+  adduser -u 1009 -Ds /bin/sh -G rclone rclone && \
+  chmod +x /startup
+
+WORKDIR /data
+
+ENV XDG_CONFIG_HOME=/config
+
+# remote configs
+ENV REMOTE_NAME= \
+  REMOTE_URL= \
+  REMOTE_VENDOR= \
+  # size with suffix K,M,G,T. Default (100Mi)
+  # set 0 to disable
+  REMOTE_NEXTCLOUD_CHUNK_SIZE=100M
+
+# s3 proxy configs
+# a space separated list of options
+ENV PROXY_ARGS=""
```

Comment:
@individual-it I tried to implement this single envvar instead of individual env for each proxy option. What do you think?